src/test/java/Homework17.java

```diff
@@ -10,22 +10,47 @@
 public class Homework17 extends LoginTests {
 
     @Test
-    public static void addSongToPlaylist() {
+    public void addSongToPlaylist() {
+
+        LoginValidEmailValidPasswordTest();
```

Comment:
**This is not a good idea to link up a couple of tests at the same time.**  If you have dependent tests then the outcome of one also affects the second testcase. 

**Tests should be 100% Independent!**

![image](https://user-images.githubusercontent.com/104761482/221399332-c8b12b07-5c3d-4abf-9fa2-091098e3da51.png)